possel/application.py

```diff
@@ -82,17 +82,19 @@ def get_etc_file(filename):
 
 def get_arg_parser():
```

Comment:
I would rather see it default to the values in the arg parser if a value isn't defined in the config file.